src/fparser/two/Fortran2003.py

```diff
@@ -8384,7 +8384,7 @@ def match(reader):
         return BlockBase.match(
             Program_Stmt, [Specification_Part, Execution_Part,
                            Internal_Subprogram_Part], End_Program_Stmt,
-            reader, match_names=True)
+            reader, match_names=True, strict_order=True)
```

Comment:
I know it is en existing error but could you please add a \ to the end of line 8376

Reply:
Thanks.